Tests/Form/Type/SimpleMoneyTypeTest.php

```diff
@@ -72,12 +62,33 @@ public function testGreaterThan1000Valid()
     public function testSetData()
     {
         \Locale::setDefault("fr_FR");
-        $moneyType = new SimpleMoneyType($this->pairManager, 2);
-        $form = $this->factory->create($moneyType, null, array());
+        $form = $this->factory->create($this->simpleMoneyTypeClass, null, array());
         $form->setData(Money::EUR(120));
         $formView = $form->createView();
 
         $this->assertEquals("1,20", $formView->children["tbbc_amount"]->vars["value"]);
     }
 
+    protected function getExtensions()
+    {
+        //This is probably not ideal, but I'm not sure how to set up the pair manager
+        // with different decimals for different tests in Symfony 3.0
```

Comment:
@geekdevs I haven't gotten too deep into how forms work because my experience with Symfony is building a REST JSONAPI.  Can you help me out with an example of how this would work?  I agree that the structure was more explicit, but I didn't know how to pass params to the constructor without getExtensions()